src/js/cypress/Browser.ts

```diff
@@ -74,6 +80,30 @@ export class IframeBrowser {
     select(selector: string, value: string) {
         return this.#body.find(selector).select(value);
     }
+
+    getElement(selector: string, timeout: BrowserTimeout = "sm") {
+        return cy.get(selector, { timeout: this.getTimeoutTime(timeout) });
+    }
+
+    getElementText(selector: string) {
+        return this.#body.get(selector).invoke("text");
+    }
+
+    clickOnText(text: string, multiple = false) {
+        return this.#body.contains(text).click({multiple});
+    }
+
+    clickFirst(selector: string, options={}) {
+        return this.#body.find(selector).first().click(options);
+    }
+
+    getElementByXpath(selector: string, timeout: BrowserTimeout = "sm") {
+        return this.#body.xpath(selector, { timeout: this.getTimeoutTime(timeout) });
+    }
+
+    getElementTextByXpath(selector: string, timeout: BrowserTimeout = "sm") {
+        return this.#body.xpath(selector, { timeout: this.getTimeoutTime(timeout) }).invoke("text");
+    }
```

Comment:
@k0stik - how can we avoid redefining the same functions for the IframeBrowser?? We should inherit the base functionality and just invoke it within the `this.#body`